lib/timeline.dart

```diff
@@ -15,19 +15,19 @@ class TimelineProperties {
   final double lineWidth;
   final double iconSize;
 
-  const TimelineProperties({Color lineColor, double lineWidth, double iconSize})
+  const TimelineProperties({Color? lineColor, double? lineWidth, double? iconSize})
       : lineColor = lineColor ?? const Color(0xFF333333),
         lineWidth = lineWidth ?? 2.5,
         iconSize = iconSize ?? TimelineBoxDecoration.DEFAULT_ICON_SIZE;
```

Comment:
this parameters should be set in the constructor as defaults, therefore should not be nullable.